fastlane/lib/fastlane/actions/git_submodule_update.rb

```diff
@@ -13,20 +13,27 @@ def self.run(params)
       #####################################################
 
       def self.description
-        "Executes a git submodule command"
+        "Executes a git submodule update command"
       end
 
       def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :recursive,
-           description: "Should the submodules be updated recursively",
-           type: Boolean,
-           default_value: false),
+                                       description: "Should the submodules be updated recursively?",
+                                       type: Boolean,
+                                       is_string: false,
```

Comment:
`is_string` can be removed 😊 `type` overrides `is_string` and `is_string` is starting to get phased out